examples/seq2seq/finetune.py

```diff
@@ -421,6 +420,9 @@ def main(args, model=None) -> SummarizationModule:
     if not args.do_predict:
         return model
 
+    if trainer.accelerator_backend:
+        trainer.accelerator_backend.barrier()
```

Comment:
what does this do?

Reply:
don't merge yet, I don't think it should live here after sleeping on it. I think this should make its way upstream! Barrier is just waiting for all distributed processes (if any) to sync (like a join).

Reply:
Sounds great, thanks!

Reply:
This has made it's way upstream now and I've pushed a commit with the lightning upgrade! Running into other failing tests, this might be the pytorch 1.7 upgrade...